spec.html

```diff
@@ -4393,6 +4393,20 @@ <h1>The Lexical Environment and Environment Record Specification Types</h1>
       <p>The Lexical Environment and Environment Record types are used to explain the behaviour of name resolution in nested functions and blocks. These types and the operations upon them are defined in <emu-xref href="#sec-lexical-environments"></emu-xref>.</p>
     </emu-clause>
 
+    <emu-clause id="sec-abstract-closure">
+      <h1>The Abstract Closure Specification Type</h1>
+      <p>The <dfn>abstract closure</dfn> specification type is used to refer to algorithmic steps together with a collection of values. Abstract closures are meta-values and are invoked using function application style such as _closure_(_arg1_, _arg2_). Like abstract operations, invocations perform the algorithmic steps described by the abstract closure. Unlike abstract operations, abstract closures are meta-values and must be created.</p>
+      <p>In algorithmic steps that create an abstract closure, values are captured with the verb "capture" followed by a list of aliases. The value associated with each alias at that time is captured. In algorithmic steps describing calling an abstract closure, captured values are referred to by the alias that was used to capture the value.</p>
```

Comment:
Should we bother to mention "captures nothing"?

Reply:
I like the consistency of always specifying what is captured, personally.

Reply:
The latter, and I think I agree.

Reply:
I think so, in the same way we'd say "takes no parameters".

Reply:
No clarification added.